tests/main/snap-quota-journal/task.yaml

```diff
@@ -62,7 +62,10 @@ execute: |
   MATCH "^LogsDirectory=$" < /etc/systemd/system/systemd-journald@snap-group-one.service.d/00-snap.conf
   
   # Now ask for the namespace logs and lets see if we can see messages from 'LogDaemon'
+  # verify the working directory of the service as well because of #LP2080332
+  retry -n 5 sh -c "$TESTSTOOLS/journal-state get-log --namespace snap-group-one | MATCH '/var/snap/test-snapd-journal-quota/x1'"
   retry -n 5 sh -c "$TESTSTOOLS/journal-state get-log --namespace snap-group-one | MATCH LogDaemon"
+  retry -n 5 sh -c "$TESTSTOOLS/journal-state get-log --namespace snap-group-one | MATCH 'logger was started'"
```

Comment:
bit surprised that it works, I was expecting the logs to go the default namespace since systemd isn't doing its mount-fu anymore, so there shouldn't be anything mounted on top of /run/log/journal, or /dev/log

Reply:
discussed on MM, there's a separate mount setting up redirection for the log namespace: https://github.com/canonical/snapd/blob/49b1224d2aef896a7c754e87fe0fed75f1ddf908/overlord/ifacestate/handlers.go#L60-L64